resources/views/components/productlist.blade.php

```diff
@@ -4,16 +4,16 @@
     <lazy v-slot="{ intersected }">
         <listing
             v-if="intersected"
-            {{-- TODO: Extract this somewhere? --}}
-            :index="config.index_prefix + '_product_' + config.store"
+            :index="config.index.product"
             v-cloak
-            v-slot="{ loaded, index, searchClient }"
+            v-slot="{ loaded, index, searchClient, middlewares }"
         >
             <div>
                 <ais-instant-search
                     v-if="searchClient"
                     :search-client="searchClient"
                     :index-name="index"
+                    :middlewares="middlewares"
```

Comment:
Can't we assign this somewhere globally? In this PR I see this 3 times including other duplication on those places. Can we extract this somewhere nicely?

Reply:
We can, and we do. most middlewares exist in the `useInstantsearchMiddlewares` store.
However if we want to add a middleware specifically for the Listing (Or in the case of this PR, the Autocomplete) we can push that in the component itself.